### ISSUE TITLE
Simplify GitHub Actions inherited from UW Hackweeks

### ISSUE THREAD

**Issue by scottyhq** [CryoInTheCloud/CryoCloudWebsite#70]
A lot of the current GitHub Actions are based un the uwhackweek JupyterBook template:
https://github.com/uwhackweek/jupyterbook-template/blob/31716111301ed43519232507742815169ad5f3f6/README.md

This includes a lot of complexity around building a linked JupyterBook and a fancy landing page. But since this repository is only concerned with building a JupyterBook and not executing notebooks a lot of the actions are not used (https://github.com/CryoInTheCloud/CryoCloudWebsite/blob/main/.github/workflows/netlifypreview.yaml ) and can be simplified or removed

**Comment by weiji14** [comment]
Yes, and a lot of the docker build related GitHub Actions can be removed too since they have been moved to https://github.com/CryoInTheCloud/hub-image. Initiall we had everything together (Jupyter Book build, docker build, etc) as mentioned at https://github.com/CryoInTheCloud/CryoCloudWebsite/issues/1#issuecomment-1305708410 to simplify things, but we've since split things into dedicated repos so there's definitely some duplicated stuff that can be removed!

On the Jupyter Book live-rendering vs pre-rendering, I see a lot of PRs tagged with `preview`, but it doesn't look like the Netlify Preview is actually set up properly to use the netlifypreview.yaml Action? Maybe @MattF-NSIDC's tutorial at https://mfisher87.github.io/posts/website-repo-pr-preview could be useful :wink: Will defer this to @tsnow03 who has done more work under the https://github.com/CryoInTheCloud/CryoCloudWebsite/tree/main/book folder, and would know if live rendering in PRs is needed.
